frontend/public/locales/en/public.json

```diff
@@ -413,6 +413,15 @@
   "LimitRanges": "LimitRanges",
   "Chargeback": "Chargeback",
   "CustomResourceDefinitions": "CustomResourceDefinitions",
+  "Build <1></1> {conjugateBuildPhase(phase)}": "Build <1></1> {conjugateBuildPhase(phase)}",
```

Comment:
I'm sorry that was not my expectation, this does not allow the i18n team to translate the phase / status message.